examples/confluent_cloud_example/confluent_cloud_example.go

```diff
@@ -182,9 +182,10 @@ func createTopic(topic string) {
 
 	// Check for specific topic errors
 	for _, result := range results {
-		if result.Error.Code() != kafka.ErrNoError {
+		if result.Error.Code() != kafka.ErrTopicAlreadyExists {
```

Comment:
It needs to check ErrNoError and ErrTopicAlreadyExists :)

Reply:
Can both situation happen? Like, fail because the topic is already there but still... cause an unexpected error during the process? (i.e.: network call error?)

Reply:
There are three possible error states that we care about here:
 1. ErrNoError - topic was created - ✅ 
 2. ErrTopicAlreadyExists - topic already exists - ✅ 
 3. Any other error - fatal error - 👎 

So this if statement should be something like: `if ..Code() != kafka.ErrNoError && ..Code() != ..TopicAlreadyExists { fail.. `

Reply:
Never mind, you're right. In the event of an error (as long this error is not TopicAlreadyExists) we should exit the program because the producer and the consumer will fail anyway.